src/steamship/client/client.py

```diff
@@ -104,6 +106,46 @@ def embed_and_search(
             space=space,
         )
 
+    def use(
+        self, app_handle: str, handle: str = None, config: Dict[str, Any] = None
+    ) -> AppInstance:
+        """Creates or loads an instance named `handle` of App named `app_handle`."""
+        instance = AppInstance.create(
+            self,
+            app_handle=app_handle,
+            handle=handle,
+            config=config,
+            upsert=True,  # TODO(ted): Rename `upsert` globally to something semantically appropriate
+        )
+
+        if instance.error:
+            raise instance.error
+        if not instance.data:
```

Comment:
Ah, yes! Will change.

Reply:
(just checked)  -- in the current definition it seems like if `response.status` is None and `response.data` is None, it will happily return `None`. That might be unlikely, but this final check will make sure it will truly never return `None` for this particular function.

I think permitting response to be `None` friendly is actually a feature, not a bug -- so leaving that check here rather than pushing it down into the `.data` accessorΩ